extension/site/matricula/browser/matricula_popup_search.mjs

```diff
@@ -0,0 +1,205 @@
+/*
+MIT License
+
+Copyright (c) 2020 Robert M Pavey
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+import {
+  addMenuItem,
+  addMenuItemWithSubMenu,
+  addBackMenuItem,
+  beginMainMenu,
+  endMainMenu,
+  doAsyncActionWithCatch,
+} from "/base/browser/popup/popup_menu_building.mjs";
+
+import { doSearch, registerSearchMenuItemFunction, shouldShowSiteSearch } from "/base/browser/popup/popup_search.mjs";
+
+import { options } from "/base/browser/options/options_loader.mjs";
+
+const matriculaStartYear = 1500;
+const matriculaEndYear = 2000;
+
+function shouldShowSearchMenuItem(data, filter) {
+  const siteConstraints = {
+    startYear: matriculaStartYear,
+    endYear: matriculaEndYear,
+    dateTestType: "bmd",
+    countryList: [],
```

Comment:
If you put "Germany" in the countryList it will mean the search menu item only shows up on the main menu when the current page has a place in Germany. Note this relies on it identifying Deutschland as Germany. I sent you a message on Discord about that.